core/modules/contact/contact.module

```diff
@@ -302,9 +302,10 @@ function contact_form_user_profile_form_alter(&$form, &$form_state) {
   $config = config('contact.settings');
   $form['contact'] = array(
     '#type' => 'fieldset',
-    '#title' => t('Contact settings'),
+    '#title' => t('Contact form'),
```

Comment:
The change seems optional. I'm guessing it was changed because the only setting within this fieldset was whether to enable the "personal contact form".